resources/assets/less/base.less

```diff
@@ -4,6 +4,14 @@
 // main: compiled.less
 @import "app";
 
+* {
+  scroll-padding-top: @nav2-height--pinned;
```

Comment:
`scroll-padding-top` only needs to be applied to `html` (and still doesn't work properly in safari)

Reply:
Hmm, I thought safari already full support it in version 14.1 (MacOS) and 14.5 (iOS) because I saw it in MDN :thinking:

Reply:
`scroll-padding` and `scroll-margin` are getting fixed in different versions for some reason 👀

Reply:
Hmm, right, Only `scroll-margin-top` currently have full support.

Should we use `scroll-margin-top` instead in `html` selector and override it locally if needs offset? Currently only one place that need offset.